doc/man/8/ceph.rst

```diff
@@ -361,19 +361,87 @@ fs
 
 Manage cephfs file systems. It uses some additional subcommands.
 
+Subcommand ``add_data_pool`` adds an new data pool to the FS. Ths pool can
+be used for file layouts as an alternate location to store the file data.
+
+Usage::
+
+    ceph fs add_data_pool <fs-name> <pool name/id>
+
+Subcommand ``authorize`` creates a new client that will be authorized for the
+given path in ``<fs_name>``. Pass ``/`` to authorize for the entire FS.
+``<perms>`` below can be ``r``, ``rw`` or ``rwp``.
+
+Usage::
+
+    ceph fs authorize <fs_name> client.<client_id> <path> <perms> [<path> <perms>...]
+
+Subcommand ``dump`` displays the FSMap at the given epoch (default: current).
+This includes all file system settings, MDS daemons and the ranks they hold
+and list of standby MDS daemons.
+
+Usage::
+
+    fs dump [epoch]
+
+Subcommand ``feature ls`` lists all CephFS features support by current
+version of Ceph.
+
+Usage::
+
+    ceph fs feature ls
+
+Subcommand ``flag set`` sets a global CephFS flag. Right now the only flag
+is ``enable_multiple`` which allows multiple CephFSs on a Ceph cluster.
+
+Usage::
+
+    ceph fs flag set <flag-name> <flag-val> --yes-i-really-mean-it
+
+Subcommand ``get`` displays the information about FS, including settings and
+ranks. Information printed here in subset of same information from the
+``fs dump`` command.
+
+Usage::
+
+    ceph fs get <fs-name>
+
 Subcommand ``ls`` to list file systems
 
 Usage::
 
 	ceph fs ls
 
+Subcommand ``lsflags`` displays all the flags set on the given FS.
+
+Usage::
+
+    ceph fs lsflags <fs-name>
+
 Subcommand ``new`` to make a new file system using named pools <metadata> and <data>
 
 Usage::
 
 	ceph fs new <fs_name> <metadata> <data>
 
-Subcommand ``reset`` is used for disaster recovery only: reset to a single-MDS map
+Subcommand ``rename`` assigns a new name to CephFS and also updates
+application tags on the pools of this CephFS.
+
+Usage::
+
+    ceph fs rename <fs-name> <new-fs-name> {--yes-i-really-mean-it}
+
+Subcommand ``required_client_features`` disables a client that doesn't
+possess a certain feature from connecting. This subcommand has two
+subcommands, one to add a requirement and other to remove the requirement.
+
+Usage::
+
+    fs required_client_features <fs name> add <feature-name>
+    fs required_client_features <fs name> rm <feature-name>
+
+Subcommand ``reset`` is used for disaster recovery only: reset to a single-MDS
+map
```

Comment:
This change just adds "map" to the new line. This change looks like can be removed?

Reply:
never mind, looks like its anyways intended to be below `rename` and `required_client_features`